installerscript.iss

```diff
@@ -0,0 +1,47 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+#define MyAppName "USFM Converter"
+#define MyAppVersion "1.0.0"
+#define MyAppPublisher "Wycliffe Associates"
+#define MyAppURL "https://wycliffeassociates.org"
```

Comment:
@danparisd aren't we publishing from Bible Translation Tools?

Reply:
Yes, I'm going to leave the publisher in case we decide to sign the app in the future, but I updated the website to bibletranslationtools.org